modules/cudf/src/column/stream_compaction.cpp

```diff
@@ -59,4 +64,17 @@ Napi::Value Column::drop_nans(Napi::CallbackInfo const& info) {
   return drop_nans(NapiToCPP(info[0]).operator rmm::mr::device_memory_resource*());
 }
 
+ObjectUnwrap<Column> Column::drop_duplicates(cudf::null_equality nulls_equal,
+                                             rmm::mr::device_memory_resource* mr) const {
+  return Column::New(std::move(
+    cudf::drop_duplicates(
+      cudf::table_view{{*this}}, {0}, cudf::duplicate_keep_option::KEEP_FIRST, nulls_equal, mr)
+      ->release()[0]));
+}
```

Comment:
I think lately the practice has been to wrap libcudf calls in `try...catch` to convert any c++ exceptions into  `NAPI_ERROR` e.g. 

https://github.com/rapidsai/node-rapids/blob/95c80d6cf655b8c6e7b33d662c2f3356850e4edd/modules/cudf/src/column/filling.cpp#L47-L56

but I also notice the existing functions don't seem to. @trxcllnt should this error-handling pattern be extended here?

Reply:
Yeah we should wrap the calls. If libcudf throws an error here, it'll lead to process abort unless we catch and rethrow as a JS exception.

Reply:
updated in c6feecc

Reply:
As long as this file is being modified I think it would be good to update the other functions above that are missing `try...catch` at the same time. But if not let's make an issue about it so it is not forgotten.